src/components/Elements/Basic.tsx

```diff
@@ -50,25 +54,34 @@ const Basic: FunctionComponent<Props> = (props) => {
       data-altid={altId}
       className={createClasses("rt-element", classes)}
       style={style}
+      data-tooltip-id="rt-tooltip"
       {...buildDataAttributes(dataSet)}
     >
       <div className="rt-element__content" aria-hidden="true">
         <span className="rt-element__title" style={titleStyle}>{title}</span>
         {continuing || <></>}
       </div>
-      <div className="rt-element__tooltip" style={tooltipStyle}>
-        {tooltip || (
+      {tooltip || (
+        <Tooltip
+          className={tooltipStyle ? "" : "rt-element__tooltip"}
```

Comment:
Why are we removing the className if there's `tooltipStyle` passed in? `tooltipStyle` should be able to overwrite the existing CSS of the classname since it's more specific.

Also, I don't think the pipeline runs the test suite when PRs are made, but can you also run the test suite to make sure removing the className doesn't have any repurcussions?